package.json

```diff
@@ -32,7 +32,7 @@
     "browser-info": "0.4.0",
     "classnames": "2.2.3",
     "clipboard": "1.5.10",
-    "cnvs": "1.1.11",
+    "cnvs": "1.1.14",
```

Comment:
We need to run `npm run shrinkwrap` once we've also bumped reactjs-components.

Reply:
@MatApple when I run `npm run shrinkwrap` locally I get some errors and I'm reluctant to fix them for fear of breaking something.

Reply:
Fixed.